Lib/test/clinic.test.c

```diff
@@ -1612,12 +1612,14 @@ test_Py_ssize_t_converter
     a: Py_ssize_t = 12
     b: Py_ssize_t(accept={int}) = 34
     c: Py_ssize_t(accept={int, NoneType}) = 56
+    d: Py_ssize_t(accept={int}, allow_negative=False) = 78
```

Comment:
Could you add some `allow_negative=True` as well just to check that the code is properly generated.